v2/rust/Cargo.toml

```diff
@@ -7,4 +7,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = { version = "*", features = ["backtrace"] }
+chrono = "0.4.31"
```

Comment:
Do we need to pin this atm? And also the other dependencies below.

Reply:
By pin you probably mean pinning the major version and requiring a lower version? I was just following the recommendations here: https://doc.rust-lang.org/cargo/reference/resolver.html#recommendations